xstate-chatbot/nodejs/src/machine/service/gis-service.js

```diff
@@ -74,6 +74,17 @@ class GisService {
     formdata.append('SewerageConnection', propertyDetails.sewageConnection);
     formdata.append('PropertyTax', propertyDetails.propertyId);
     formdata.append('OwnersName', propertyDetails.ownerName);
+
+    if(propertyDetails.image){
+      let filestoreId = await this.getFileForFileStoreId(propertyDetails.image);
```

Comment:
This will NOT give you a file store id. It will return with a File object.

Reply:
gis service updated

Reply:
gis service updated again